### ISSUE TITLE
0.5b8 does not work on FF 69 Nigthly

### ISSUE THREAD

**Issue by Marcosdw** [Infocatcher/Right_Links_WE#16]
Hello,

0.5b8 does not work on FF 69 Nightly

Greetings

--
Sorry, i use an online translator French / English

**Comment by sir-harlekin** [comment]
Also not working with FF 68b3. Was working with FF67 till the update a few minutes ago.

**Comment by Marcosdw** [comment]
Yes, i confirm not working with FF 68Bx, i updated the title subject.

**Comment by sir-harlekin** [comment]
Additional info for 68b3: Long left click is still working, right click is not.

**Comment by sir-harlekin** [comment]
Working again with b9. Thank you dear Infocatcher!
